src/models/rankData.ts

```diff
@@ -0,0 +1,29 @@
+import mongoose from 'mongoose';
+
+// SCHEMA SETUP
+const rankDataSchema = new mongoose.Schema({
+    username: {
+        type: String,
+        required: true,
+    },
+    seasonNumber:{
+        type: Number,
+    },
+    playerRating: {
+        type: Number,
+        default: 1500,
```

Comment:
Let's extract these constants somewhere so they're shared with the reset command.